portfolio/src/main/webapp/index.html

```diff
@@ -8,35 +8,129 @@
     <link rel="icon" href="/images/AD.png" type="image/png" sizes="16x16">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+
+    <!-- Font Awesome icons CDN -->
+    <script src="https://use.fontawesome.com/4e51920be5.js"></script>
   </head>
 
   <body>
     <!-- Landing page -->
-    <header class="showcase">
+    <header id="home" class="showcase">
         <div class="content">
+
             <img src="images/AD.png" class="logo" alt="AD">
             <h1 class="title">
                 Anusha Datta
             </h1>
             <p class="text">
                 CODER. CREATOR. ARTIST.
             </p>
+
         </div>
+
+        <!-- Scroll arrow -->
+        <a href="#navbar"> <img src="images/icons/arrow.png" class="arrow" alt="scroll"> </a>
     </header>
 
+    <!-- Navigation Bar -->
+    <div id="navbar" class="sticky-navbar">
+        <ul>
+            <li>
+                <a href="#home">HOME</a>
+            </li>
+            <li>
+                <a href="#about">ABOUT</a>
+            </li>
+            <li>
+                <a href="#education">EDUCATION</a>
+            </li>
+            <li>
+                <a href="#home">WORK EXPERIENCE</a>
+            </li>
+            <li>
+                <a href="#home">PROJECTS</a>
+            </li>
+            <li>
+                <a href="#home">BLOG</a>
+            </li>
+            <li>
+                <a href="#contact">CONTACT</a>
+            </li>
+
+            <img src="images/AD.png" class="navbar-logo">
+        </ul>
+    </div>
+
     <!-- About -->
-    <section class="bg-light">
-        <div class="container grid-2">     
-            <div class="center">
-                <img src="images/avatar.jpg" height="300px">
+    <section id="about" class="bg-light">
+        <div class="container">
+            
+            <h1>About</h1>
+            <br>
```

Comment:
Using a lot of breakpoints might become a little unmanageable. Consider adding a style to h1 tag above which then you can re-use for similar scenarios. For eg if you want to change newline style you will have to find the breakpoint again.

Reply:
Okay yes I got you, I'll surely keep that in mind.